util/db/dbutil.go

```diff
@@ -228,6 +228,12 @@ func (db *Accessor) atomic(fn idemFn, extras ...interface{}) (err error) {
 				if !ok {
 					err = fmt.Errorf("%v", r)
 				}
+
+				buf := make([]byte, 16*1024)
+				stlen := runtime.Stack(buf, false)
+				errstr := string(buf[:stlen])
+				fmt.Printf("recovered panic in atomic: %s", errstr)
```

Comment:
@cce Asking because I'm unfamiliar with conventions - Is stderr preferable over acquiring a logger + logging error level log message?

I can make the change once agreed upon.

Reply:
Also - I'll make the change _after_ merging this PR.

Reply:
deadlock detector does `fmt.Fprintln(os.Stderr, string(buf))` that looks more correct for me

Reply:
@algorandskiy Thanks for the response - I still have some low urgency questions about when to write to stderr directly vs using a logger, but happy to bring in the requested change.

Resolving - I opened https://github.com/algorand/go-algorand/pull/4712 to address the feedback.